### PR TITLE
Feat/reward liquid assets

### PR DESCRIPTION
**Pull request by kenta-elys**
- Add rewards, liquid assets & liquid asset endpoints and states and types.

### CODE REVIEW

File: .vscode/settings.json
Comment (politeWall) [inline .vscode/settings.json:1]: .vscode is in the gitignore and should not be there

File: src/error.rs [hunk 1]
```diff
@@ -8,4 +7,6 @@ pub enum ContractError {
     StdError(#[from] StdError),
     #[error("{sender} is not the owner of the order")]
```
Comment (politeWall) [inline src/error.rs:8]: the Unauthorized error is not used in the contract

File: src/action/query/get_liquid_asset.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+use super::*;
+use crate::{bindings::query::ElysQuery, msg::query_resp::GetLiquidAssetResp, types::*, states::LIQUID_ASSETS};
+
+pub fn get_liquid_asset(deps: Deps<ElysQuery>, asset: String) -> Result<GetLiquidAssetResp, ContractError> {
+    let liquid_assets: Vec<LiquidAsset> = LIQUID_ASSETS.load(deps.storage)?;
+    let have_assets = liquid_assets.iter().find(|liquid_asset| liquid_asset.asset == asset);
+    let resp = GetLiquidAssetResp {
+        liquid_asset: match have_assets {
+            Some(liquid_asset) => liquid_asset.to_owned(),
+            None => LiquidAsset::new_dummy(),
```
Comment (politeWall) [inline src/action/query/get_liquid_asset.rs:10]: why do you return a new_dummy instead of returning  the error `LiquidAssetNotFound`

**Review by politeWall — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (politeWall) [inline .vscode/settings.json:1]: Why is there a .vscode directory if it's in the .gitignore?

**Review by cosmic-vagabond — APPROVED** [review]

**Review by politeWall — APPROVED** [review]

**Pull request merged**